xsd/NeTEx_publication.xsd

```diff
@@ -3066,17 +3171,17 @@ Provides a general purpose wrapper for NeTEx data content.</xsd:documentation>
 			<xsd:field xpath="@id"/>
 			<xsd:field xpath="@version"/>
 		</xsd:key>
-		<!-- =====Entrance============================== -->
+		<!-- =====Entrance==============================  
```

Comment:
Why is this commented out?

Reply:
looks like a bug ...

Reply:
Will assume an error, as 
`xsd:element name="Entrance" type="SiteEntrance_VersionStructure" abstract="false" substitutionGroup="SiteComponent">`
is defined in netex_part_1/part1_ifopt/netex_ifopt_site_version.xsd:337

REMARK: Have kept the comment regardless, someone (@nick-knowles ?) should review and (maybe) revert to again include the 
`<xsd:keyref name="Entrance_AnyKeyRef" refer="netex:Entrance_AnyVersionedKey">`